cfme/common/provider.py

```diff
@@ -680,9 +682,9 @@ def _fill_credential(form, cred, validate=None):
         else:
             principal = cred.principal
         fill(cred.form, {'default_principal': principal,
-            'default_secret': cred.secret,
-            'default_verify_secret': cred.verify_secret,
-            'validate_btn': validate})
+                         'default_secret': cred.secret,
+                         'default_verify_secret': cred.verify_secret,
+                         'validate_btn': validate})
```

Comment:
Are you talking about the indentation? 
When I get it the way it was before I am getting an error:
http://i.imgur.com/Mxvox7G.png